src/js/static.js

```diff
@@ -22,8 +22,8 @@ gpii.express.router["static"].init = function (that) {
     that.staticHandler = express["static"](that.options.content);
 };
 
-gpii.express.router["static"].route = function (that, req, res) {
-    that.staticHandler(req, res);
+gpii.express.router["static"].route = function (that, req, res, next) {
+    that.staticHandler(req, res, next);
```

Comment:
Having said that, it doesn't explain why it was possible in gpii-json-schema.  Will keep digging.